rusoto/services/apigateway/src/generated.rs

```diff
@@ -9,6 +9,7 @@ use std::error::Error;
 use rusoto_core::request::HttpDispatchError;
 use rusoto_core::credential::{CredentialsError, ProvideAwsCredentials};
 
+use std::str::FromStr;
```

Comment:
This generally doesn't need to be imported unless referencing the trait itself. It's simpler to use as implicitly via [`parse()`](https://doc.rust-lang.org/std/primitive.str.html#method.parse). Then there's no extra unused import! :)